piety-promoted-v2/modernized/17-chapter-16.adoc

```diff
@@ -735,14 +740,17 @@ and he had several opportunities with them to his satisfaction,
 particularly with the chief magistrate of the town.
 In a letter to a Friend he gives this account of it:
 
-'`He asked me many questions concerning our principles, which I answered short,
+[.embedded-content-document.letter]
+--
+
+He asked me many questions concerning our principles, which I answered short,
 but so full that he made no objections; and I was thankful in my mind it was so,
 for it was somewhat difficult for the young man my interpreter,
 though he is always ready to assist me when I have occasion.
 When I got home to my solitary dwelling,
 and considered how often the Lord had appeared on my behalf, and had been my advocate,
 I was much bowed in thankfulness before him.
-I much desire I may be remembered by you, '`(meaning Friends,)`' for good,
+I much desire I may be remembered by you, (meaning Friends,) for good,
```

Comment:
I'm wondering if the narrator inserted this in parantheses? idk, and idk if it matters?

Reply:
It made me wonder if it was the narrator because it was in quotation marks.

Reply:
I think it probably was added by the narrator, but I think the way you removed the quotes and left the parens seems correct. 👍